src/features/live_sensor/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const SENSOR_CONTROL = 'sensorControl';
+export const SENSOR_CONTROL_NAME = 'Live sensor control';
+
+export const UPDATES_PER_MINUTE = 60;
+// in milliseconds
+export const RECORD_UPDATES_INTERVAL = 100;
```

Comment:
both const seem not needed if you push to server based on location update

Reply:
`RECORD_UPDATES_INTERVAL ` defines how often we add data to array before sending it